suite-common/wallet-utils/src/exportTransactions.ts

```diff
@@ -115,46 +125,102 @@ const makePdf = (
+                            fiat: '', // missing rates for tokens
+                            fiatSymbol: '',
+                            other: '',
+                        };
+                    }
+
+                    return null;
+                });
+            }
+            return t.targets.map((target, index) => {
                 if (target?.addresses?.length && target?.amount) {
-                    return `${target.addresses[0]} (${target.amount}) ${
-                        target.metadataLabel ? `- ${target.metadataLabel}` : ''
-                    }`;
+                    return {
+                        ...sharedData,
+                        fee: index === 0 ? t.fee : '', // fee only once per tx
+                        feeSymbol: index === 0 ? coin.toUpperCase() : '',
+                        address: target.isAddress ? target.addresses[0] : '', // SENT - it is destination address, RECV - it is MY address
+                        label: target.isAddress && target.metadataLabel ? target.metadataLabel : '',
+                        amount: target.isAddress ? target.amount : '',
+                        symbol: target.isAddress ? coin.toUpperCase() : '',
+                        fiat:
+                            target.isAddress &&
+                            target.amount &&
+                            t.rates &&
+                            t.rates[data.localCurrency]
+                                ? Intl.NumberFormat(undefined, {
+                                      style: 'decimal',
+                                      maximumFractionDigits: 2,
+                                      minimumFractionDigits: 2,
+                                  })
+                                      .format(
+                                          parseFloat(target.amount) * t.rates[data.localCurrency]!,
+                                      )
+                                      .toString()
+                                : '',
+                        fiatSymbol: data.localCurrency,
+                        other: !target.isAddress ? target.addresses[0] : '', // e.g. OP_RETURN
+                    };
                 }
 
                 return null;
             });
-        }
-
-        return {
-            ...t,
-            addresses: addresses.join(addressSeparator[type]),
-            type: t.type.toUpperCase(),
-            datetime: new Intl.DateTimeFormat('default', dateTimeFormat).format(
-                (t.blockTime || 0) * 1000,
-            ),
-        };
-    });
+        })
+        .filter(record => record !== null) as Fields[];
 };
 
-const prepareCsv = (fields: Field, content: any[]) => {
+const prepareCsv = (data: Data) => {
+    const csvFields: Fields = {
+        timestamp: 'Timestamp',
+        date: 'Date',
+        time: 'Time',
+        type: 'Type',
+        txid: 'Transaction ID',
+        fee: 'Fee',
+        feeSymbol: 'Fee coin',
```

Comment:
Maybe `Fee Unit`? Or same pattern as with `Fiat (USD)` column or Fee in PDF? Or is it possible that some fees has different units? 
@Hannsek

Reply:
We want to have the separate Fee Unit column, because we want to be prepared for export from many different accounts…